res/skins/Tango/skin_settings.xml

```diff
@@ -43,15 +48,33 @@ Description:
             <Children>
 
               <Template src="skin:skin_settings_button_2state.xml">
-                <SetVariable name="state_0_text">      Stacked Overviews</SetVariable>
-                <SetVariable name="state_1_text"> &#10004; Stacked Overviews</SetVariable>
-                <SetVariable name="Setting">[Tango],stacked_overviews</SetVariable>
+                <SetVariable name="state_0_text">      Compact mode</SetVariable>
+                <SetVariable name="state_1_text"> &#10004; Compact mode</SetVariable>
+                <SetVariable name="Setting">[Tango],enforce_netbook_mode</SetVariable>
+              </Template>
+
+              <Template src="skin:skin_settings_button_2state.xml">
+                <SetVariable name="state_0_text">      Symmetric Overviews</SetVariable>
+                <SetVariable name="state_1_text"> &#10004; Symmetric Overviews</SetVariable>
+                <SetVariable name="Setting">[Tango],symmetric_overviews</SetVariable>
+              </Template>
+
+              <Template src="skin:skin_settings_button_2state.xml">
+                <SetVariable name="state_0_text">      Symmetric Time/Duration</SetVariable>
+                <SetVariable name="state_1_text"> &#10004; Symmetric Time/Duration</SetVariable>
+                <SetVariable name="Setting">[Tango],symmetric_time</SetVariable>
+              </Template>
+
+              <Template src="skin:skin_settings_button_2state.xml">
+                <SetVariable name="state_0_text">      Symmetric Time/Duration</SetVariable>
+                <SetVariable name="state_1_text"> &#10004; Symmetric Time/Duration</SetVariable>
+                <SetVariable name="Setting">[Tango],symmetric_time</SetVariable>
               </Template>
 
               <Template src="skin:skin_settings_button_2state.xml">
-                <SetVariable name="state_0_text">      Star Rating</SetVariable>
-                <SetVariable name="state_1_text"> &#10004; Star Rating</SetVariable>
-                <SetVariable name="Setting">[Tango],stars</SetVariable>
+                <SetVariable name="state_0_text">      4 &#9664;  8 Hotcues</SetVariable>
```

Comment:
The idea was to design it like a physical flip switch: both states are clear no matter which is active.
Do you insist on changing it to `4 Hotcues` and `8 Hotcues`?